test/e2e/e2e.sh

```diff
@@ -161,14 +160,27 @@ function e2e::run() {
   local testcases
   IFS=" " read -r -a testcases <<<"$(e2e::list_test_cases)"
   local total_cnt="${#testcases[@]}"
+  OPEN_KRUISE_ENABLED_IN_RISINGWAVE=0
 
   logging::infof "Running tests, total %d...\n" "${total_cnt}"
 
   local cur_cnt=0
   local pass_cnt=0
   local fail_cnt=0
   # shellcheck disable=SC2155
-  local begin_ts=$(date +%s)
+  # local begin_ts=$(date +%s)
+  # for tc in "${testcases[@]}"; do
+  #   if e2e::test::run_next "${cur_cnt}" "${tc}"; then
+  #     ((pass_cnt++))
+  #   else
+  #     ((fail_cnt++))
+  #   fi
+  #   ((cur_cnt++))
+  # done
+
+  logging::info "Running tests for when openkruise is enabled"
+  OPEN_KRUISE_ENABLED_IN_RISINGWAVE=1
+  testenv::k8s::risingwave_operator::enable_openkruise
```

Comment:
Looks like the `OPEN_KRUISE_ENABLED_IN_RISINGWAVE` is forcefully overwritten here. Is it by expectation or a mistake?

Reply:
Have amended!